packages/connectivity/example/macos/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Flutter-related
```

Comment:
Same comment here about removing the .gitignore.

Reply:
Done